pubsub/google/cloud/pubsub_v1/subscriber/policy/base.py

```diff
@@ -346,13 +346,17 @@ def maintain_leases(self):
             _LOGGER.debug('Snoozing lease management for %f seconds.', snooze)
             time.sleep(snooze)
 
-    def modify_ack_deadline(self, ack_id, seconds):
+    def modify_ack_deadline(self, ack_id, seconds=None):
```

Comment:
I would prefer this be explicitly sent to this method, not have it be a default.